tests/prove_payment_tests.move

```diff
@@ -0,0 +1,117 @@
+#[test_only]
+module bitcoin_spv::prove_payment_tests;
+
+use bitcoin_spv::params;
+use bitcoin_spv::light_client::{new_light_client_with_params_int, ETxNotInBlock};
+use bitcoin_spv::transaction::{make_transaction, parse_output};
+use sui::test_scenario;
+
+
+#[test]
+fun prove_payment_test() {
```

Comment:
```suggestion
fun test_prove_payment() {
```

Reply:
lets change it in the entire file. This is the naming convention we follow in other tests, lets be consistent